cieidsdk/src/main/java/it/ipzs/cieidsdk/common/CieIDSdk.kt

```diff
@@ -127,7 +140,7 @@ object CieIDSdk : NfcAdapter.ReaderCallback {
 
     private fun checkCodiceServer(codiceServer: String): Boolean {
         val regex = Regex("^[0-9]{16}$")
-        if(regex.matches(codiceServer)){
+        if (regex.matches(codiceServer)) {
```

Comment:
```suggestion
        return regex.matches(codiceServer)
```
we could return the result directly